### ISSUE TITLE
The tab order of the document detailed view (pro interface) should be: get / description

### ISSUE THREAD

**Issue by iGormilhit** [rero/rero-ils#1078]
**How it works**

The tab order of the document detailed view of the professional interface is *description* and *get*.

**How it should works**

The tab order of the document detailed view of the professional interface should be *get* and *description*, as it is on the public interface.
